### PR TITLE
Add gstshark-plot filter option

### PR DESCRIPTION
**Pull request by mleivaf**

### CODE REVIEW

File: scripts/graphics/gstshark-plot [hunk 1]
```diff
@@ -20,6 +20,9 @@ then
     echo "                                 over each graphic generated"
     echo "                                 extern: display the legend over an external window"
     echo "                                 (default: inside)"
+    echo " -f, --filter [descriptor]       Filter events that have the word provided. For example"
+    echo "                                 Select one element information: $0 gstshark_filtered -p --filter x264enc"
+    echo "                                 Select more the one element:    $0 gstshark_filtered -p --filter \"x264enc|udpsink\""
```
Comment (michaelgruner) [inline scripts/graphics/gstshark-plot]: is this some standard regex syntax? if so, can we specify so?

Comment (mleivaf) [reply scripts/graphics/gstshark-plot]: it Is a grep sintax, is an option to allow an "or" filter

Comment (michaelgruner) [reply scripts/graphics/gstshark-plot]: Ok I see you use `grep -E`, that’s extended regex. Please specify this, so users can know which syntax is valid

Comment (mleivaf) [reply scripts/graphics/gstshark-plot]: Done

**Pull request merged**